docs/interactions/Message_Components.md

```diff
@@ -14,16 +14,20 @@ The top-level `components` field is an array of [Action Row](#DOCS_INTERACTIONS_
-| emoji?      | partial [emoji](#DOCS_RESOURCES_EMOJI/emoji-object)                           | `name`, `id`, and `animated`                                                        | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons)         |
-| custom_id?  | string                                                                        | a developer-defined identifier for the button, max 100 characters                   | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons)         |
-| url?        | string                                                                        | a url for link-style buttons                                                        | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons)         |
-| disabled?   | boolean                                                                       | whether the button is disabled, default `false`                                     | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons)         |
-| components? | array of [components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object) | a list of child components                                                          | [Action Rows](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/action-rows) |
+| Field        | Type                                                                                                        | Description                                                                         | Valid For                                                                                                                    |
+| ------------ | ----------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| type         | integer                                                                                                     | [component type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-types)             | all types                                                                                                                    |
+| custom_id?   | string                                                                                                      | a developer-defined identifier for the button, max 100 characters                   | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons), [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus) |
+| disabled?    | boolean                                                                                                     | whether the button is disabled, default `false`                                     | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons), [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus) |
```

Comment:
```suggestion
| custom_id?   | string                                                                                                      | a developer-defined identifier for the component, max 100 characters                | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons), [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus) |
| disabled?    | boolean                                                                                                     | whether the component is disabled, default `false`                                  | [Buttons](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons), [Select Menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus) |
```

Since these aren't just for buttons

Reply:
+1 to this